python/pyproject.toml

```diff
@@ -16,7 +16,7 @@ whylogs-sketching = ">=3.4.1.dev3"
 protobuf = ">=3.19.4"
 importlib-metadata = { version = "<4.3", python = "<3.8" }
 typing-extensions = { version = ">=3.10", markers = "python_version < \"4\""}
-whylabs-client = "^0.6.5"
+whylabs-client = "^0.6.15-dev0"
```

Comment:
Do we want to leave this as a dev dependency?

Reply:
nope, I'll update this now and push again. Just a timing issue with the platform release

Reply:
@richard-rogers just updated the version